Makefile

```
@@ -7,32 +7,12 @@ MAKEFLAGS += --no-builtin-variables
-	@echo "  integration-test - run integration tests"
-	@echo "  fmt         - format code"
-	@echo "  vet         - run go vet"
-	@echo "  install     - install binary"
-	@echo "  clean       - clean build artifacts"
-	@echo "  dist        - create release"
-	@echo "  sandbox    - run sandbox container"
-
-all: build test
-	@echo "Run 'make check' for full checks including integration tests"
+.PHONY: check build build-fzf build-all test test-cov integration-test fmt install clean dist dist-tui dist-fzf tag sandbox test-sup-versions docker-hub vet setup-env golangci-lint
 
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)